index.bs

```diff
@@ -35,18 +35,26 @@ for cross-site identifiers like third-party cookies.
 
 ## Overview ## {#overview}
 
-A page can register an [=attribution source=] on a site by providing
- <{a/attributionsourceeventid}> and <{a/attributiondestination}> attributes on an <{a}> element.
-When such an <{a}> element is clicked, and the resulting navigation commits in a document within the [=same site=] as
-the <{a/attributiondestination}>, the [=attribution source=] is stored in UA storage.
-
-At a later point, the <{a/attributiondestination}> site may fire an HTTP request to
-trigger attribution, which matches an [=attribution trigger=] with any previously
-stored sources. If a matching source exists, it is scheduled to be
-reported at a later time, possibly multiple days in the future.
-
-Reports are sent to reporting endpoints that are configured in the attribution source
-and attribution trigger.
+Pages/embedded sites are given the ability to register [=attribution sources=] and 
+[=attribution triggers=] can be linked by the User Agent to generate and 
+send [=attribution reports=] which contain information from both of those
```

Comment:
s/which contain/containing

Otherwise this is feeling like a very run on sentence